frontend/src/selectors.ts

```diff
@@ -47,3 +47,4 @@ export const selectTaxonsByID = (state: RootState) => state.taxons.itemsByID as
 export const selectToken = (state: RootState) => state.auth.tokens.access
 export const selectUsersByID = (state: RootState) => state.users.itemsByID as ItemsByID<User>
 export const selectWorkflowsByID = (state: RootState) => state.workflows.itemsByID as ItemsByID<Workflow>
+export const selectCoordinatesByID = (state: RootState) => state.coordinates.itemsByID as ItemsByID<Coordinate>
```

Comment:
I'm trying to keep these in alphabetical order (although I see I missed one :) )